src/gen_idna_table.c

```diff
@@ -101,7 +107,8 @@ static void generate_mappings()
            "        return NULL;\n"
            "    int low = 0, high = %d;\n"
            "    while (low < high) {\n"
-           "        int middle = (4 * low + high) / 5;\n" /* heuristic low bias */
+           "        int middle = (4 * low + high) / 5;\n" /* heuristic low bias
+                                                           */
```

Comment:
This looks weird. But not a really an issue.

Reply:
Fixed by embedding the comment in the generated code.